OpenRA.Mods.Common/Traits/SpawnActorOnDeath.cs

```diff
@@ -29,11 +28,13 @@ public class SpawnActorOnDeathInfo : ConditionalTraitInfo
 		public readonly int Probability = 100;
 
 		[Desc("Owner of the spawned actor. Allowed keywords:" +
-			"'Victim', 'Killer' and 'InternalName'.")]
+			"'Victim', 'Killer' and 'InternalName'. " +
+			"Falls back to 'InternalName' if 'Victim' is used " +
+			"and the victim is defeated (see 'SpawnAfterDefeat').")]
 		public readonly OwnerType OwnerType = OwnerType.Victim;
 
 		[Desc("Map player to use when 'InternalName' is defined on 'OwnerType'.")]
-		public readonly string InternalOwner = null;
+		public readonly string InternalOwner = "Creeps";
```

Comment:
Why Creeps?  IMO Neutral would be a more intuitive default.

Reply:
RA2 spawned to Creeps.

Reply:
In OpenRA creeps are considered hostile and will target (and more importantly for husks etc be targeted *by*) all players.

Reply:
By the principle of least surprise our defaults should be as boring as possible, and modders should have to opt in to any additional/surprising behaviour.  Neutral fits that bill much better than Creeps, and justifying creeps by saying that Husks turn off the creep specific behavior reinforces that point.